packages/ng/core-select/input/select-input.component.ts

```diff
@@ -242,6 +254,22 @@ export abstract class ALuSelectInputComponent<TOption, TValue> implements OnDest
 		setTimeout(() => this.focusInput());
 	}
 
+	shouldDisplayAddOption(): Observable<boolean> {
+		switch (this.addOptionStrategy) {
+			case 'always':
+				return of(true);
+			case 'never':
+				return of(false);
+			case 'if-empty-clue':
+				return this.clue$.pipe(map((clue) => !!clue));
+		}
+	}
```

Comment:
Shouldn't we make this an Observable rather than a method and have the strategy be a subject? That way we could properly handle option strategy change at runtime.

Reply:
I hesitated!